components/kyma-environment-broker/internal/event-hub/azure/namespace_test.go

```diff
@@ -0,0 +1,23 @@
+package azure
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/Azure/azure-sdk-for-go/services/eventhub/mgmt/2017-04-01/eventhub"
+	"github.com/Azure/go-autorest/autorest"
+)
+
+func TestGetResourceGroup(t *testing.T) {
+
+	resourceGroup := "my-resourcegroup"
+	eventHubNamespaceID := fmt.Sprintf("/subscriptions/35d42578-34d1-486d-a689-012a8d514c19/resourceGroups/%s/providers/Microsoft.EventHub/namespaces/nachtmaar-test-2", resourceGroup)
+	nameSpaceFixture := eventhub.EHNamespace{
+		Response: autorest.Response{},
+		ID:       &eventHubNamespaceID,
+	}
+	parsedResourceGroup := GetResourceGroup(nameSpaceFixture)
+	if parsedResourceGroup != resourceGroup {
```

Comment:
you can just use `assert.Equal(t, resourceGroup, parsedResourceGroup)` from "github.com/stretchr/testify/assert" library

Reply:
Method is not required anymore. Thanks for the suggestion 👍